arangod/Scheduler/SchedulerFeature.cpp

```diff
@@ -103,10 +103,9 @@ void SchedulerFeature::validateOptions(
 
   if (_nrMaximalThreads == 0) {
     _nrMaximalThreads = 4 * _nrServerThreads;
-  }
-
-  if (_nrMaximalThreads < 64) {
-    _nrMaximalThreads = 64;
+    if (_nrMaximalThreads < 64) {
+      _nrMaximalThreads = 64;
+    }
```

Comment:
The above change, and its original, is completely confusing.  If I have a two core processor, do I really want 64 threads all the time?  If so, just add a comment.

Reply:
This is the definitive upper limit. `--server.maximal-threads` is a hidden option which is basically only needed for debugging. Our unfortunately not very asynchronous server infrastructure makes it currently necessary that in extreme situations we run a lot of threads. The SchedulerManager dynamically starts new Scheduler threads as needed, but never more than _nrMaximalThreads.Therefore, I believe that the "minimum" of 64 for `_nrMaximalThreads` is still sensible for this reason. However, if we give `--server.maximal-threads` with a small value, we mean it. Therefore this should not be overwritten by 64, which would make testing with few threads essentially impossible. So the *only* change here is to *not* enforce at least 64 for this variable if it was explicitly specified by a user in the know about this option.